src/main/java/frc/robot/subsystems/Leds.java

```diff
@@ -44,19 +48,22 @@ private boolean isBlueTeam() {
         }
     }
 
-    public Command setTeamColor() {
-        // The command is named "Sample Command" and requires this subsystem.
+    public Command setTeamColor(List<Integer> v) {
```

Comment:
Replace this with `public Command setTeamColor(Integer... v)`, then down below instead of creating an iterator, do `for(Integer x : v) {`

The way that this is set up is alright, but requires that `Arrays.asList` call above.

Reply:
Yup.. That's a much cleaner way to do it. I'll have @Rohan-Sreenivasan update these at our next meeting.